exchanges/exchanges_test.go

```diff
@@ -5,6 +5,8 @@ package exchanges
 
 import (
 	"context"
+	"fmt"
+	// "encoding/json"
```

Comment:
Leftover from test data generation?

Reply:
I just use it a lot for inspection. No need to keep it though.